src/command.test.js

```diff
@@ -116,6 +116,46 @@ describe('Command', () => {
           expect(outcome.errors.input1[0][0]).toEqual('invalid')
         })
       })
+      describe('.addInputErrorAndHalt()', () => {
+        class InputHaltCommand extends Command {
+          static schema =
+          {
+            coolestInput: { type: 'string' }
+
```

Comment:
does the linter run here?

Reply:
As we mentioned on huddle. For some reason this is allowed for `standard.js` 🤷‍♂️